apps/tunnel/src/main.c

```diff
@@ -273,23 +286,20 @@ static bool tunnel_parse_args(int argc, char* argv[], nabto_main_setup* nms) {
         nms->localPort = localPort;
     }
 
-
-    if (gopt(options, 's')) {
-        const char* preSharedKey;
-        uint8_t key[16];
-        memset(key, 0, 16);
-        if ( gopt_arg( options, 'k', &preSharedKey)) {
-            if (!unabto_read_psk_from_hex(preSharedKey, nms->presharedKey ,16)) {
-                NABTO_LOG_FATAL(("Cannot read psk"));
-            }
+    const char* preSharedKey;
```

Comment:
Can we add an option for that such that we have the possibility for debugging crypto performance problems with the standard executable?

it could be a long option like --unencrypted